.changelog/17511.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:fix
```

Comment:
I would remove the changelog entry since you are making a modification to an unreleased feature.  So basically this is a change that customers won't need to know about.  Basically, the whole feature should get one changelog entry for the feature being released rather than several smaller updates with every PR.

Please remove this entry and add the `pr/no-changelog` label to the PR.